includes/downloads/recalculations.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Recalculation functions for downloads.
+ * @package     EDD
+ * @subpackage  Downloads
+ * @copyright   Copyright (c) 2022, Easy Digital Downloads, LLC
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ * @since       3.1
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * The hook is registered here because it's specifically for the cron job,
+ * but it calls the general download recalculation function.
+ */
+add_action( 'edd_recalculate_download_sales_earnings_deferred', 'edd_recalculate_download_sales_earnings' );
+
+/**
+ * Handles scheduling the recalculation for a download.
+ *
+ * @since 3.1
+ * @param int $download_id
+ * @return void
+ */
+function edd_maybe_schedule_download_recalculation( $download_id ) {
+	$is_scheduled = wp_next_scheduled( 'edd_recalculate_download_sales_earnings_deferred', array( $download_id ) );
+	$bypass_cron  = apply_filters( 'edd_recalculate_bypass_cron', false );
+	if ( $is_scheduled && ! $bypass_cron ) {
+		edd_debug_log( 'Recalculation is already scheduled for ' . $download_id . ' at ' . edd_date_i18n( $is_scheduled, 'datetime' ) );
+		return;
+	}
```

Comment:
```suggestion
	}

```
This just feels 'cramped', can we give these 3 if statements a little breathing room?